src/test/java/core/basesyntax/service/impl/FileWriterServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.exception.FileWritingException;
+import core.basesyntax.service.FileWriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileWriterServiceImplTest {
+    private static final String VALID_REPORT = "fruit,quantity\n"
+            + "banana,20";
+    private static final String VALID_TEST_REPORT_PATH = "src/test/resources/testReport.csv";
+    private static final FileWriterService fileWriterService = new FileWriterServiceImpl();
+
+    @AfterClass
+    public static void clearFile() {
+        try {
+            Files.writeString(Path.of(VALID_TEST_REPORT_PATH), "");
+        } catch (IOException e) {
+            throw new RuntimeException("Couldn't delete the report data: "
+                    + VALID_TEST_REPORT_PATH, e);
+        }
+    }
```

Comment:
"AfterClass" methods are usually placed at the end of the class

Reply:
fixed